docs/src/installing.rst

```diff
@@ -3,30 +3,26 @@
 Installing
 ==========
 
-Iris is available using conda for the following platforms:
-
-* Linux 64-bit,
-* Mac OSX 64-bit, and
-* Windows 64-bit.
-
-Windows 10 now has support for Linux distributions via WSL_ (Windows
-Subsystem for Linux).  This is a great option to get started with Iris
-for users and developers.  Be aware that we do not currently test against
-any WSL_ distributions.
-
-.. _WSL: https://learn.microsoft.com/en-us/windows/wsl/install
+Iris can be installed using conda or pip.
 
 .. note:: Iris is currently supported and tested against |python_support|
           running on Linux.  We do not currently actively test on other
           platforms such as Windows or macOS.
 
+          Windows 10 now has support for Linux distributions via WSL_ (Windows
+          Subsystem for Linux).  This is a great option to get started with
+          Iris for users and contributors.  Be aware that we do not currently
+          test against any WSL_ distributions.
```

Comment:
Moved this down here as I think the fact that Iris is only tested on Linux motivates use of WSL, but I may have got the wrong end of the stick.  I've never attempted to do anything with python on Windows...